metricbeat/mb/mb_test.go

```diff
@@ -375,3 +375,26 @@ func newConfig(t testing.TB, moduleConfig interface{}) *common.Config {
 	}
 	return config
 }
+
+func Test_QueryParams_String(t *testing.T) {
```

Comment:
This name doesn't follow the style of the file.

Reply:
I didn't realized that there was some style, sorry :( I'm not sure about a proper name here, maybe `TestModuleConfigQueryParams`?

Reply:
SGTM

Reply:
Fixed in 3890c8a440bdb3302525788e70dfb221a9feb6b0